### PR TITLE
Relax active support dependency

### PR DESCRIPTION
**Pull request by sobrinho**

### CODE REVIEW

**Comment by sobrinho** [comment]
Depends on #1 being merged first.

File: ebayr.gemspec [hunk 1]
```diff
@@ -27,8 +27,8 @@ command-line client which aids integration into other projects.
     gem.add_development_dependency 'minitest'
   else
     gem.add_dependency 'nokogiri', '~> 1.6'
-    gem.add_dependency 'activesupport', '~> 4.0'
+    gem.add_dependency 'activesupport', '>= 4.0', '< 5.1'
```
Comment (oehlschl) [inline ebayr.gemspec]: We're upgrading to 5.1 (and then more), so maybe < 6 is better.

**Review by oehlschl — CHANGES_REQUESTED** [review]
See comment

**Comment by oehlschl** [comment]
@sobrinho were you able to assert that specs pass in a Rails 5 env?

**Comment by sobrinho** [comment]
@oehlschl yes, the specs are passing:

```
$ grep activesupport Gemfile.lock
      activesupport (>= 4.0, < 6)
    activesupport (5.1.3)

$ bundle exec rake test
MiniTest::Unit::TestCase is now Minitest::Test. From /Users/sobrinho/Developer/ebayr/test/ebayr/record_test.rb:6:in `<module:Ebayr>'
/Users/sobrinho/Developer/ebayr/lib/ebayr/request.rb:84: warning: shadowing outer local variable - http
/Users/sobrinho/Developer/ebayr/lib/ebayr/request.rb:114: warning: mismatched indentations at 'end' with 'case' at 111
/Users/sobrinho/Developer/ebayr/lib/ebayr.rb:145: warning: File.exists? is a deprecated name, use File.exist? instead
/Users/sobrinho/Developer/ebayr/lib/ebayr.rb:145: warning: File.exists? is a deprecated name, use File.exist? instead
/Users/sobrinho/Developer/ebayr/lib/ebayr.rb:145: warning: File.exists? is a deprecated name, use File.exist? instead
/Users/sobrinho/Developer/ebayr/lib/ebayr.rb:145: warning: File.exists? is a deprecated name, use File.exist? instead
Run options: --seed 41926

# Running:

..........................

Finished in 0.401172s, 64.8101 runs/s, 291.6455 assertions/s.

26 runs, 117 assertions, 0 failures, 0 errors, 0 skips
```

I tested against 4, 5.0 and 5.1.

**Comment by sobrinho** [comment]
To be clear, I tested against active support 4, 5.0 and 5.1, not rails because the gem doesn't depend on rails itself, just the AS.

**Review by oehlschl — APPROVED** [review]

**Pull request merged**